### ISSUE TITLE
ImpulseNoise error with TypeError: 'ReplaceElementwise' object is not callable

### ISSUE THREAD

**Issue by YubinXie** [aleju/imgaug#308]
For the same image,

```images_aug = iaa.ImpulseNoise(p=0.1)(images=images)``` gives error:
 TypeError: 'ReplaceElementwise' object is not callable

While this works:
```
noise = iaa.ImpulseNoise(p=0.1)
images_aug = noise.augment_images(images)
```
Many examples in the tutorial are in the first form. Is there a difference in usage between different function? thanks!

![B19D5EBE-AD82-46D5-8502-F6430FDF97C7](https://user-images.githubusercontent.com/16257776/57345278-40185100-7118-11e9-913d-45591434c2c7.png)

**Comment by aleju** [comment]
Which version are you using? In 0.2.9 the form `ImpulseNoise(...)(images=...)` should work, but you might still be on 0.2.8.

`augment_images()` does essentially the same as `augment(images=...)`.

**Comment by YubinXie** [comment]
You are right. I was using 0.2.8. It is fixed after upgrading to 0.2.9. Thanks!
